addons/web/static/src/views/kanban/animated_number.js

```diff
@@ -5,7 +5,7 @@ import { formatInteger } from "@web/views/fields/formatters";
 
```

Comment:
We separated and renamed elements to avoid the Activity view to import kanban named stuff. We were not fan of the fact that this view was depending/inheriting so much of the kanban view. We wanted a more generic component shared by those views.

But should we keep this component in the kanban folder or move it elsewhere?

Reply:
i think it's fine to keep it here for now. maybe will be moved later, but no real need to do it now